Initial draft of GCP Archive documentation (#604)
vespa-engine/cloud: en/archive-guide-gcp.md

---
# Copyright Yahoo. All rights reserved.
title: GCP Archive guide
layout: page
---

Vespa Cloud exports log data, heap dumps, and Java Flight Recorder sessions to
buckets in Google Cloud Storage. This guide explains how to access this data.

> Access to the data must happen through GCP project controlled by the tenant.

Reviewer: ```suggestion
Access to the data is through a GCP project controlled by the tenant.
```

Data traffic to access this data is charged to this GCP project.

These resources are needed to get started:
* A GCP project
* A Google user account
* The [gcloud command line interface](https://cloud.google.com/sdk/docs/install)

Access is configured through the Vespa Cloud Console in the tenant account screen.  Choose
the "archive" tab, then "GCP" tab to see the settings below.

## Register IAM principal
![Register IAM  principal](../assets/archive-1-gcp.png)

> First, a principal must be granted access to the Cloud Storage bucket in Vespa
> Cloud. This is done by entering a [principal](https://cloud.google.com/iam/docs/overview) with a supported prefix. See the accepted format in the description below the input field. 

Reviewer: Sentences, or semi-sentences, are easier reviewed and later changed if written like one would write code, one per line - e.g.
```suggestion
First, a principal must be granted access to the Cloud Storage bucket in Vespa Cloud.
This is done by entering a [principal](https://cloud.google.com/iam/docs/overview) with a supported prefix.
See the accepted format in the description below the input field. 
```
